src/legacy/core_plugins/kibana/migrations.js

```diff
@@ -55,7 +59,42 @@ function migrateIndexPattern(doc) {
     });
   }
   doc.attributes.kibanaSavedObjectMeta.searchSourceJSON = JSON.stringify(searchSource);
-}
+};
+
+// [TSVB] Migrate percentile-rank aggregation (value -> values)
+const migratePercentileRankAggregation = doc => {
```

Comment:
The problem with this function name is, what if in `8.0.0` we want to add an 8.x migration that migrates percentile rank aggregations? What would we name it? I think this logic would either be better inlined the way the `7.0.0` migration was, or else this function should probably somehow indicate which migration version it belongs to: `migratePercentileRankAggregation_7_1` or something.